webpack.config.production.js

```diff
@@ -1,14 +1,24 @@
 var webpack = require('webpack');
+var InlineManifestWebpackPlugin = require('inline-manifest-webpack-plugin');
 var config = require('./webpack.config.base');
 
 config.bail = true;
 config.profile = false;
 config.devtool = 'source-map';
+config.output.filename = '[name].[chunkhash].js';
```

Comment:
I was going to say I didnt think you needed the chunkhash since html-webpack-plugin handles adding the hash, but I see that hashing wasn't ever turned on in the boilerplate *sigh*